### ISSUE TITLE
Radius not working

### ISSUE THREAD

**Issue by cristianditomasso** [jaagr/polybar#1643]
## Describe the issue
Can't get round/radius borders to work.
**Expected behavior:**
Round/radius borders to work

**Was it working before?**
First time using polybar.

## To Reproduce
**[bar/test]
monitor = HDMI-A-0
width = 85%
height = 31
offset-x = 7%
offset-y = 1%
radius = 15.0
fixer center = true
override-redirect = true
wm-restack = i3

line-size = 2
line-color = #933957

;border-size = 2
;border-color = #f7f3f2** 

``

## Screenshots
https://imgur.com/a/w3hi9mq

## Environment:
* WM: i3

**Comment by patrick96** [comment]
Hmm, I think border-radius only works when you have a compositor running or have set `pseudo-transparency = true` in the `settings` section. Are you running a compositor like compton?

**Comment by cristianditomasso** [comment]
> Hmm, I think border-radius only works when you have a compositor running or have set `pseudo-transparency = true` in the `settings` section. Are you running a compositor like compton?

YES!

Adding pseudo-transparency = true to the settings section fixed my issue. Thanks a lot!

Compton wasn't needed at all by the way :)

**Comment by patrick96** [comment]
Yeah, you either need pseudo-transparency or compton. I have added a note to the wiki :)
